deepseg.cc

```diff
@@ -76,6 +76,29 @@ cv::Mat convert_rgb_to_yuyv( cv::Mat input ) {
 	return yuyv;
 }
 
+cv::Mat alpha_blend(cv::Mat srca, cv::Mat srcb, cv::Mat mask) {
```

Comment:
Isn't there something for this already in OpenCV?

Reply:
Nothing obvious: it has a constant alpha blend function `addWeighted`, and masked binary blending via `copyTo` or `add` each with a mask parameter.
There is the usual half answered SO question: https://stackoverflow.com/questions/41263010/how-to-blend-images-using-a-mask-in-opencv and an unanswered OpenCV forum one: https://answers.opencv.org/question/28439/blending-two-images-with-masked-roi/

Reply:
Searching their "new" forums I stumbled upon:
https://forum.opencv.org/t/opencv-java-how-to-superimpose-three-images-that-have-the-same-size/857/2

This links to this blog post:
https://learnopencv.com/alpha-blending-using-opencv-cpp-python/

Reply:
Heh - literally where I got my code from..